lib/logstash-docket/util/alias_plugin_util.rb

```diff
@@ -0,0 +1,20 @@
+# encoding: utf-8
+
+require "yaml"
+require "net/http"
+
+module LogstashDocket
+  module Util
+    ##
+    # A util class defines repetitive logics for aliased plugins.
+    #
+    class AliasPluginUtil
+
+      ALIAS_MAPPINGS_URL = 'https://raw.githubusercontent.com/elastic/logstash/master/logstash-core/src/main/resources/org/logstash/plugins/AliasRegistry.yml'
+
+      def fetch_alias_mappings
+        YAML::safe_load(Net::HTTP.get(URI(ALIAS_MAPPINGS_URL))) || {}
```

Comment:
The `|| {}` fall-back to empty mapping is at best overly "safe" and at worst will allow the processing to silently fail to load aliases if (and only if?) the mappings is fetched but contains an empty YAML. I would prefer this to fail loudly if we can't load the mapping instead of producing a subset of the desired content.

```suggestion
        YAML::safe_load(Net::HTTP.get(URI(ALIAS_MAPPINGS_URL))) || fail('empty alias mapping')
```

Reply:
Makes sense, applied!